WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Summary/SimplePaymentsSummaryViewModel.swift

```diff
@@ -267,6 +273,21 @@ final class SimplePaymentsSummaryViewModel: ObservableObject {
 
 // MARK: Helpers
 private extension SimplePaymentsSummaryViewModel {
+    /// Strips the currency symbol from the formatted amount
+    ///
+    func removeCurrencySymbolFromAmount(_ amount: String) -> String {
```

Comment:
We could also move this extension to the CurrencyFormatter as a static function, I was was 50/50 on doing so but ultimately just left it here since we do not use it anywhere else in the app, happy to iterate further if needed.

Reply:
Yeah, I agree that it feels more natural there